### ISSUE TITLE
Weak Raycast crash

### ISSUE THREAD

**Issue by FoxMcloud5655** [Kamefrede/rpsideas#27]
It seems as though the Weak Raycast crashes the game.  There's also a spelling error in the description.

Log:
```
java.lang.NullPointerException: Unexpected error
	at vazkii.psi.api.spell.SpellPiece.getParamValue(SpellPiece.java:135)
	at com.kamefrede.rpsideas.util.helpers.SpellHelpers.getNumber(SpellHelpers.java:74)
	at com.kamefrede.rpsideas.util.helpers.SpellHelpers.getBoundedNumber(SpellHelpers.java:80)
	at com.kamefrede.rpsideas.spells.selector.PieceMacroCasterWeakRaycast.execute(PieceMacroCasterWeakRaycast.java:31)
```

**Comment by Kamefrede** [comment]
Fixed in 2.1.2

**Comment by Kamefrede** [comment]
sorta, I forgot to fix the typo :( that'll be fixed in 2.2.0
